.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: publish
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
All I need to do to get this to work after this is push:

1. Ask TeamHelp to create an NPM account for automated deployments under the npm repository
2. Ask them to create an access token
3. Add the access token to Buildertrend org's secrets (someone with org permissions will need to do this)
  1. Configure so that the secret is passed down to this specific repo

Then to publish,
1. Make a commit that updates the `package.json` version
2. Create a release in GitHub

Reply:
@C-Hess - Has this been done, or do we need @rkalasky or @dteske25 on this?

Should SREs own this? Or are we good with having the open-sourced stuff live outside of that?

Reply:
This has not be done yet. In my opinion, I think it's okay for Devs to take care of so as not to require more "cooks in the kitchen" so to speak.

Reply:
Sounds good. I vote we wait to merge until this setting is setup

Reply:
The setting is in! Though we won't _really_ be able to test this until we actually need to do a publish (which should be somewhat soon as I have some ESLint tickets to look into)